### ISSUE TITLE
Create 0.2.4 release

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/scpca-nf#128]
Most of the docs now refer to version 0.2.4, as there were some updates in the workflow related to that.

I have created draft release notes in https://github.com/AlexsLemonade/scpca-nf/releases/edit/untagged-b3cfb48ee17dede68328, which I think cover the changes that have been made.

After that release is done, the 0.2.4 draft should be merged into the `latest` branch so that the `defaultBranch` tag we added in the manifest works as expected. 

Tagging @allyhawkins in case there are any additional tasks that she can think of before we complete this release.

**Comment by allyhawkins** [comment]
I believe we have addressed everything that needs to go in before we make a release. I did do some more testing of the config file today on slurm to be sure that it was set up correctly since the minor changes in #121 and it appears to run up until the last step of the workflow (generating the filtered RDS file) and then I am getting a permission error which looks like something specific to my slurm setup. Just wanted to note that somewhere.
We should be good to go on the release and the notes look good to me!
